### ISSUE TITLE
What would be the license?

### ISSUE THREAD

**Issue by brunobord** [QuinnFreedman/quick-quest#1]
Hello,

First of all, thank you very much for this lovely (almost-)one-pager!

A quick question that would probably has to be worked out with the original author of the ruleset (/u/giffyglyph/), but what would be the license of these documents? By default, they're "all rights reserved" (which is fair, you do whatever you want with your intellectual property).
But what if somebody wanted to make a derivative of this work? (a hack, a translation, advanced or optional rules, more spells, etc)

Maybe that'd be great to specify the license explicitly, in order to know if it's okay for you to allow derivative works or not, and upon which conditions.

**Comment by QuinnFreedman** [comment]
I am personally fine with anyone using them or modifying them in any way. I uploaded the html I used to make them so that they can be modified. I would probably put them in the public domain if I had full ownership.

I'm not really sure how the law works here, though. I copied content from another user that didn't have any license I could find, and the adventure is heavily derivative from a 3rd-party article published in a TSR product. I guess it's possible I'm already in violation by publishing my version.

It's probably safe to do what I did and just fork it without worrying about license. If you wanted to sell it you'd probably have to contact the original copyright holder(s), though. I'll look into if there is a standard license that expresses this idea.
